samples/subsys/lorawan/class_a/README.rst

```diff
@@ -25,3 +25,16 @@ The following commands build and flash the sample.
    :board: nucleo_wl55jc
    :goals: build flash
    :compact:
+
+Important Notes for Multiple Runs
+*********************************
+
+By default, this example works only once with the current configuration. On subsequent attempts, the device may fail to join the LoRaWAN network due to a hardcoded ``dev_nonce`` value. According to the LoRaWAN specification, ``dev_nonce`` must increment for every new connection attempt. To ensure the sample works on multiple runs, follow these recommendations:
```

Comment:
On subsequent attempts, the LoRaWAN network server may reject the join request due to ...